api_server/src/request/mod.rs

```diff
@@ -276,7 +275,7 @@ mod tests {
         check_error_response(vmm_resp, StatusCode::InternalServerError);
         let vmm_resp = VmmActionError::StartMicrovm(
             ErrorKind::Internal,
-            StartMicrovmError::CreateBlockDevice(sys_util::Error::new(22)),
+            StartMicrovmError::CreateBlockDevice(std::io::Error::from_raw_os_error(22)),
```

Comment:
nit: since `std::io` is already `use`d:
```suggestion
            StartMicrovmError::CreateBlockDevice(io::Error::from_raw_os_error(22)),
```

Reply:
done